samples/04_gis_analysts_data_scientists/calculate_impervious_surfaces_from_spectral_imagery.ipynb

```diff
@@ -84,7 +84,7 @@
     "from arcgis.gis import GIS\n",
     "from arcgis.raster.functions import RFT           \n",
     "\n",
-    "gis = GIS(url='https://pythonapi.playground.esri.com/portal', username='arcgis_python', password='amazing_arcgis_123')"
+    "gis = GIS('https://pythonapi.playground.esri.com/portal')"
```

Comment:
this issue is because RA is not configured properly. you could try even by passing credentials and verify it that makes it work.

In RA samples, what i observed is that .save() is mainly failing. probably raster analytics tool might also fail